usefull-test-data/zipcodes.md

```diff
@@ -0,0 +1,134 @@
+usefull addresses
+---
+**NOTE**
+The description for each of these type of addresses is in dutch. For a list with English descriptions it better to visit
+ - packages//mocking/src/zipcode.ts 
+ - packages/mocking/src/restriction.ts
+ 
+---
+
+### Een huisnummer dat voorkomt in de toevoeging (1 én 1A, 1B, 1C):
```

Comment:
Why is this part in dutch?

Reply:
I don't mind that this part is Dutch. This is for internal testing purposes only. It would be a lot of work to translate just for the sake of translating. And it would make it more harder to read.